src/components/victory-candlestick/candle.js

```diff
@@ -0,0 +1,98 @@
+import React, { PropTypes } from "react";
+// import d3Shape from "d3-shape";
+
+
+export default class Candle extends React.Component {
+  static propTypes = {
+    index: React.PropTypes.number,
+    x: PropTypes.number,
+    wickY1: PropTypes.number,
+    wickY2: PropTypes.number,
+    candleColor: PropTypes.string,
+    candleY: PropTypes.number,
+    candleWidth: PropTypes.number,
+    candleHeight: PropTypes.number,
+    scale: PropTypes.object,
+    datum: PropTypes.object
+  }
+
+  render() {
+    return (
+      <g>
+        <Wick
```

Comment:
since these two components are so simple and so related it might make sense to just have functions inside `Candle` that return these svg elements rather than separate components.